### PR TITLE
feat: interval per-probe

### LINKED ISSUE #25: Interval should be per-probe
Right now, interval configuration is made global, and probes were executed in series within that global interval.

The interval should be per-probe, where each probe have their own interval.

### PR DESCRIPTION
**Pull request by kevinhermawan**
This PR fixes #25

### CODE REVIEW

File: src/utils/looper.ts [hunk 1]
```diff
@@ -1,76 +1,78 @@
...
+    log(`Probe Interval: ${probe.interval}`)
+    log(`Probe Request Method: ${probe.request.method}`)
+    log(`Probe Request URL: ${probe.request.url}`)
+    log(`Probe Request Headers: ${JSON.stringify(probe.request.headers)}`)
+    log(`Probe Request Body: ${JSON.stringify(probe.request.body)}`)
+    log(`Probe Alerts: ${probe.alerts.toString()}\n`)
+
+    const probeInterval = setInterval(async () => {
+      return doProbe(probe, config.notifications)
+    }, (probe.interval ?? 0) * MILLISECONDS)
```
Comment (nicnocquee) [inline src/utils/looper.ts:72]: I think default value is better be 10 seconds. Please put the default value as const at the beginning of the file.

File: src/utils/looper.ts [hunk 2]
```diff
@@ -1,76 +1,78 @@
...
-        probe: item,
-        validatedResp,
-        trueThreshold: item.trueThreshold || 5,
-        falseThreshold: item.falseThreshold || 5,
-      })
+    const serverStatuses = processProbeStatus({
+      probe,
+      validatedResp,
+      trueThreshold: probe.trueThreshold || 5,
+      falseThreshold: probe.falseThreshold || 5,
```
Comment (nicnocquee) [inline src/utils/looper.ts:30]: Please put the default value as const at the top.

File: config.example.json [hunk 3]
```diff
@@ -49,9 +48,10 @@
       "id": "1",
       "name": "Example",
       "description": "Probe",
+      "interval": 0,
```
Comment (nicnocquee) [inline config.example.json:51]: Change the value in this example to 10.

File: docs/src/pages/quick-start.md [hunk 4]
```diff
@@ -28,6 +27,7 @@ Follow the following steps to quickly monitor this Monica website and to get not
          "id": "1",
          "name": "Monika Landing Page",
          "description": "Landing page of awesome Monika",
+         "interval": 10,
```
Comment (nicnocquee) [inline docs/src/pages/quick-start.md:30]: Please add explanation about this interval in the docs.

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Review by sapiderman — APPROVED** [review]
looks good!

**Review by dennypradipta — APPROVED** [review]

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**